website/src/react-native/index.js

```diff
@@ -67,7 +67,7 @@ var App = React.createClass({
             All operations between the JavaScript application code and the native platform are performed asynchronously, and the native modules can also make use of additional threads as well.  This means we can decode images off of the main thread, save to disk in the background, measure text and compute layouts without blocking the UI, and more.  As a result, React Native apps are naturally fluid and responsive.  The communication is also fully serializable, which allows us to leverage Chrome Developer Tools to debug the JavaScript while running the complete app, either in the simulator or on a physical device.
           </p>
           <p>
-            See <a href="docs/debugging.html">Debugging</a>.
+            See <a href="docs/debugging.html#content">Debugging</a>.
           </p>
```

Comment:
Without this change, when clicking on the link on mobile you get sent to the Debugging page but scrolled to the top of the page. Because the menu system goes first and is long, it can seem like the page is broken or the content is missing. 

Ultimately the site just needs better mobile support and links shouldn't have to require #content everywhere. But alas, that's where we are now.